parser.cs

```diff
@@ -777,8 +785,29 @@ public class Parser {
 
     private void optionalActualParameterList(){
```

Comment:
Tried to do a := b * c and it didn't work. Then tried a := b * 5 which also didn't work. So, either µPascal sucks hard, or we did something wrong.